CMakeLists.txt

```diff
@@ -227,7 +227,7 @@ install(
 )
 
 # Headers are all combined into one common 'include' dir when MOOS is installed
-set(MOOS_INCLUDE_DIRS "include")
+set(MOOS_INCLUDE_DIRS "\${CMAKE_CURRENT_LIST_DIR}/../../../include")
```

Comment:
A cleaner way of doing this is with file(RELATIVE_PATH ...) and this is really something that should be calculated inside the MOOSConfig.cmake file rather than in the top level CMakeLists s/t it is relocatable.